File: NERBABO.Frontend/src/app/shared/enums/approval-status.enum.ts
export enum ApprovalStatus {
  NotSpecified = 'Não Especificado',
  Approved = 'Aprovado',
  Rejected = 'Reprovado'
}

export const ApprovalStatusOptions = [
  { label: ApprovalStatus.NotSpecified, value: ApprovalStatus.NotSpecified },
  { label: ApprovalStatus.Approved, value: ApprovalStatus.Approved },
  { label: ApprovalStatus.Rejected, value: ApprovalStatus.Rejected }
];

File: NERBABO.Backend/NERBABO.ApiService/Shared/Enums/ApprovalStatusEnum.cs
using System.ComponentModel;

namespace NERBABO.ApiService.Shared.Enums;

/// <summary>
/// This enumeration defines the different approval status options for student action enrollments.
/// It includes options ranging from not specified to approved and rejected.
/// Use Humanizer to get the description of each enum value.
/// </summary>
public enum ApprovalStatusEnum
{
    [Description("Não Especificado")]
    NotSpecified,

    [Description("Aprovado")]
    Approved,

    [Description("Reprovado")]
    Rejected,

    [Description("Desistiu")]
    Dropped
}